src/android/com/adobe/phonegap/push/PushPlugin.java

```diff
@@ -65,21 +65,7 @@ public void run() {
 
                         Log.v(LOG_TAG, "execute: senderID=" + senderID);
 
-                        String savedSenderID = sharedPref.getString(SENDER_ID, "");
-                        String savedRegID = sharedPref.getString(REGISTRATION_ID, "");
-
-                        // first time run get new token
-                        if ("".equals(savedRegID)) {
-                            token = InstanceID.getInstance(getApplicationContext()).getToken(senderID, GCM);
-                        }
-                        // new sender ID, re-register
-                        else if (!savedSenderID.equals(senderID)) {
-                            token = InstanceID.getInstance(getApplicationContext()).getToken(senderID, GCM);
-                        }
-                        // use the saved one
-                        else {
-                            token = sharedPref.getString(REGISTRATION_ID, "");
-                        }
+                        token = FirebaseInstanceId.getInstance().getToken();
```

Comment:
On my sample project I have this string on res/values/strings.xml (created by the plugin) `<string name="google_app_id">myAppId3333</string>` and I'm able to receive push notifications sent from FCM console, but I have nothing for gcm_defaultSenderId anywhere